scripts/in_container/install_airflow_and_providers.py

```diff
@@ -603,6 +603,17 @@ def install_airflow_and_providers(
     use_distributions_from_dist: bool,
     install_airflow_with_constraints: bool,
 ):
+    if mount_sources in ["tests", "remove"]:
```

Comment:
Reversing removal of installed "editable" packages to happen before installing with `--use-airflow-version`